src/Pkg.jl

```diff
@@ -5,8 +5,14 @@ import REPL
 using REPL.TerminalMenus
 
 depots() = Base.DEPOT_PATH
-logdir() = joinpath(depots()[1], "logs")
-devdir() = get(ENV, "JULIA_PKG_DEVDIR", joinpath(depots()[1], "dev"))
+function depots1()
+    d = depots()
+    isempty(d) && cmderror("no depots found in DEPOT_PATH")
```

Comment:
maybe this should just be a regular error? Or I guess this will make more sense after #475